server/publications/collections/products.js

```diff
@@ -496,3 +503,35 @@ Meteor.publish("Products", function (productScrollLimit = 24, productFilters, so
     // limit: productScrollLimit
   });
 });
+
+/**
+ * @name Products/grid
+ * @method
+ * @memberof Core
+ * @summary Publication method for a customer facing product grid
+ * @param {number} productScrollLimit - product find limit
+ * @param {object} productFilters - filters to be applied to the product find
+ * @param {object} sort - sorting to be applied to the product find
+ * @return {MongoCursor} Mongo cursor object of found products
+ */
+Meteor.publish("Products/grid", function (productScrollLimit = 24, productFilters, sort = {}) {
```

Comment:
Why do we use a capital `P` here as opposed to lowercase?

Reply:
I think it was to match the name of the `Products` publication and collection name.